biomage/utils/data.py

```diff
@@ -18,13 +77,57 @@ def modified_records(item, target_table, config):
             "meta": {
                 "M": {
                     **item["meta"]["M"],
-                    "pipeline": {
+                    "gem2s": {
                         "M": {
-                            "stateMachineArn": {"S": ""},
-                            "executionArn": {"S": ""},
+                            **item["meta"]["M"]["gem2s"]["M"],
+                            "paramsHash": {"S" : ""},
```

Comment:
why do you need to empty it if you are going to replace it? because the hash itself would change with this value?

Reply:
No reason haha. I'll remove this.